### DISCUSSION TITLE
docs/linux/gaming/superflight/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#4] [Giscus]
# docs/linux/gaming/superflight/

Image stolen from https://store.steampowered.com/app/732430/Superflight/
Cool game introduced to me by my friend Nate. The game lags a bit but is overall pretty playable on Linux.
Run this launch option:
PROTON_USE_WINED3D=1 %command% 

https://liluzibird.github.io/docs/linux/gaming/superflight/

**Comment by armando013005** [comment]
can i play it in androind?

**Reply by liluzibird** [reply]
Nah. It was on sale during the steam winter sale. I think there's an android [APK](https://apkcombo.com/superflight-simulator/com.SuperF.SuperflightSimulator) unofficially that may or may not give you viruses.
